pkg/antctl/raw/supportbundle/command.go

```diff
@@ -300,13 +301,22 @@ func createAgentClients(k8sClientset kubernetes.Interface, antreaClientset antre
 		if !ok {
 			continue
 		}
-		ip, err := k8s.GetNodeAddr(&node)
+		ips, err := k8s.GetNodeAddrs(&node)
 		if err != nil {
 			klog.Warningf("Error when parsing IP of Node %s", node.Name)
 			continue
 		}
 		cfg := rest.CopyConfig(cfgTmpl)
-		cfg.Host = net.JoinHostPort(ip.String(), port)
+		var nodeIP string
+		if ips.IPv4 != nil {
+			nodeIP = ips.IPv4.String()
+		} else if ips.IPv6 != nil {
```

Comment:
According to https://github.com/antrea-io/antrea/pull/2436/files#r686106932, I think you don't need to check 3 cases, just if else can cover all cases?

Reply:
Make sense. Updated.